packages/ui-icons/docs/keywords.json

```diff
@@ -5841,7 +6613,8 @@
       "search"
     ],
     "category": "Generic",
-    "release": "3.19.1"
+    "release": "3.19.1",
+    "codepoint": ""
```

Comment:
Might be better to use `null` for the ones that don't have a codepoint value. Otherwise, we're mixing strings and numbers, whereas `null` represents no value. Applies to all 'empty' `codepoint` fields.